src/app/home/problem-list/problem-list.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@angular/core';
+import { AngularFirestore } from '@angular/fire/firestore';
+import { Problem } from 'src/app/home/problem-list/problem.model';
+
```

Comment:
Let's create a new `service` folder at the level of `app`. Within that folder we can keep a `http-services` folder and a `utility-services` folder. Put the service files and specs under there.

Reply:
Created service folders and moved` problem-list-service` and its spec to `utility-services`
[Commit link](https://github.com/armory-uci/vital/pull/8/commits/066d6038cd7ea68ebd6bcc242bd6fed17dc374fb)